ci-operator/step-registry/baremetalds/assisted/conf/baremetalds-assisted-conf-commands.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "************ baremetalds assisted conf command ************"
+
+echo "export NODES_PLATFORM=assisted" >> ${SHARED_DIR}/dev-scripts-additional-config
+echo "export IP_STACK=v4" >> ${SHARED_DIR}/dev-scripts-additional-config
+echo "export NUM_WORKERS=1" >> ${SHARED_DIR}/dev-scripts-additional-config
+echo "export INSTALL_OPERATOR_SDK=0" >> ${SHARED_DIR}/dev-scripts-additional-config
+echo "export MANAGE_BR_BRIDGE=n" >> ${SHARED_DIR}/dev-scripts-additional-config
+
+echo "assisted" >> ${SHARED_DIR}/makefile-target
```

Comment:
Do we really a new file just for that?

Reply:
Is there another way to transfer data between steps?

Reply:
The `${SHARED_DIR}` mechanism is the only one available. My point was about trying to reuse `dev-scripts-additional-config`, but it's not a blocking one